src/DynamoUtilities/AssemblyHelper.cs

```diff
@@ -34,20 +34,29 @@ public Assembly ResolveAssembly(object sender, ResolveEventArgs args)
             try
             {
                 var targetAssemblyName = new AssemblyName(args.Name).Name + ".dll";
-
+                Console.WriteLine("trying to resolve");
```

Comment:
some logging - we could get rid of this.

Reply:
I would think it's fine to leave it if not creating too messy logging. Do you have an example of the message?

Reply:
``` Trying to resolve 
CoreNodeModels
at
some/path ```

```
``` Trying to resolve 
CoreNodeModels
at
some/path ```

```
```loading from
some\path]\CoreNodeModels
 ```